mix.exs

```diff
@@ -45,6 +45,7 @@ defmodule Habits.Mixfile do
       {:gettext, "~> 0.11.0"},
       {:cowboy, "~> 1.0"},
       {:comeonin, "~> 2.0"},
+      {:secure_random, "~> 0.5"},
```

Comment:
I probably don't need a library for this (it's [pretty simple](http://stackoverflow.com/a/32002566/995663)) but like to avoid reinventing the wheel where security is concerned.